src/interfaces.ts

```diff
@@ -173,6 +174,8 @@ export interface ICoreKit {
   /**
    * Enable MFA for the user. Deletes the Cloud Factor and generates a new Factor Key.
    * Recommended for Non Custodial Flow.
+   * @param enableMFAParams - Parameters for recovery factor for MFA.
+   * @param recoveryFactor - Default is true. If false, recovery factor will NOT be created.
    */
   enableMFA(enableMFAParams: EnableMFAParams): Promise<string>;
```

Comment:
You are adding a parameter to the function doc without adding the parameter to the interface...